packages/core/src/components/forms/_control-group.scss

```diff
@@ -215,6 +215,42 @@ Styleguide components.forms.control-group
   }
 
   /*
+  Responsive control groups
+
+  Add the class `pt-fill` to a control group to make all controls expand equally to fill the
```

Comment:
please note that the inverse is also possible: adding `.pt-fill` to individual controls, which you do immediately in the first control group in the markup below.